hibernate-orm/core/Spatial/Spatial.gradle

```diff
@@ -7,12 +7,12 @@ repositories {
         url "http://www.hibernatespatial.org/repository"
     }
     maven {
-        url "http://repo.opengeo.org"
+        url "http://repo.boundlessgeo.com/main"
     }
 }
 
 dependencies {
 	compile( 'org.hibernate:hibernate-spatial:4.3-SNAPSHOT')
```

Comment:
4.3-SNAPSHOT?

As you volunteered, you should probably upgrade to a more recent version :).

Reply:
I don't know if I want to go so deep into the demo.
I'll update to a 4.3   Final for now

Reply:
Well, let's merge this one and let the ORM guys update the demos :).